### PR TITLE
generate and upload torrent files

### PR DESCRIPTION
**Pull request by nirajpathak13**
Torrent files are available here for testing

 https://s3.console.aws.amazon.com/s3/buckets/download-chia-net?region=us-west-2&prefix=torrents/&showversions=false

### CODE REVIEW

File: .github/workflows/build-linux-arm64-installer.yml [hunk 1]
```diff
@@ -105,13 +105,24 @@ jobs:
...
 
+    - name: Install py3createtorrent
+      if: startsWith(github.ref, 'refs/tags/')
+      run: |
+        pip3 install py3createtorrent
+
+   - name: Create torrent
+     if: startsWith(github.ref, 'refs/tags/')
+     run: |
+       py3createtorrent -t torrent.chia.net ${{ github.workspace }}/build_scripts/final_installer/chia-blockchain_${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}_arm64.deb ${{ github.workspace }}/build_scripts/final_installer/chia-blockchain_${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}_arm64.deb.torrent
```
Comment (arvidn) [inline .github/workflows/build-linux-arm64-installer.yml:116]: I would imagine we'd also want to add the URL where we host this file too, as a `--webseed`

Comment (nirajpathak13) [reply .github/workflows/build-linux-arm64-installer.yml:116]: Yeah, this was actually just a test run, this part of the automation is going to be moved to a different project.

File: azure-pipelines.yml [hunk 2]
```diff
@@ -100,7 +100,7 @@ steps:
       ls $(System.DefaultWorkingDirectory)/build_scripts/final_installer/
       cd $(System.DefaultWorkingDirectory)/build_scripts/
       export CHIA_VERSION="Chia-"$(<version.txt)
-      shasum -a 256 $(System.DefaultWorkingDirectory)/build_scripts/final_installer/$CHIA_VERSION.dmg > $(System.DefaultWorkingDirectory)/build_scripts/final_installer/$CHIA_VERSION.dmg.checksum
+      shasum -a 256 $(System.DefaultWo1rkingDirectory)/build_scripts/final_installer/$CHIA_VERSION.dmg > $(System.DefaultWorkingDirectory)/build_scripts/final_installer/$CHIA_VERSION.dmg.sha256
```
Comment (arvidn) [inline azure-pipelines.yml:103]: this looks like a typo

**Review by hoffmang9 — APPROVED** [review]
lgtm

**Pull request merged**